### PR TITLE
ci: schedule daily job

### PR DESCRIPTION
**Pull request by v1v**
## Motivation/summary

Dynamically trigger jobs related to the update-beats automation. This approach uses the active branches detection, therefore we won't need to do much as older branches won't trigger any extra builds.

I'll wait for the renaming branch that happens today

### CODE REVIEW

File: .ci/schedule-daily.groovy [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  triggers {
+    cron('H H(2-3) * * 1-5')
+  }
+  stages {
+    stage('Nighly update Beats builds') {
+      steps {
+        updateBeatsBuilds(quietPeriodFactor: 2000, branches: ['master', '8.<minor>', '7.<minor>', '7.<next-minor>'])
```
Comment (v1v) [inline .ci/schedule-daily.groovy:23]:
```suggestion
        updateBeatsBuilds(branches: ['master', '8.<minor>', '7.<minor>', '7.<next-minor>'])
```

Comment (v1v) [inline .ci/schedule-daily.groovy]: Aliases to know what's the current active minor version and next upcoming minor version. It relies in some of the automation we have built.

**Review by axw — APPROVED** [review]

File: .ci/schedule-daily.groovy [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+  }
+  triggers {
+    cron('H H(2-3) * * 1-5')
+  }
+  stages {
+    stage('Nighly update Beats builds') {
+      steps {
+        updateBeatsBuilds(branches: ['master', '8.<minor>', '7.<minor>', '7.<next-minor>'])
```
Comment (v1v) [inline .ci/schedule-daily.groovy:23]:
```suggestion
        updateBeatsBuilds(branches: ['main', '8.<minor>', '7.<minor>', '7.<next-minor>'])
```

File: .ci/jobs/apm-server-schedule-daily.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+- job:
+    name: apm-server-schedule-daily
+    display-name: Jobs scheduled daily (weekdays)
+    description: Jobs scheduled daily (weekdays)
+    project-type: pipeline
+    parameters:
+      - string:
+          name: branch_specifier
+          default: master
```
Comment (v1v) [inline .ci/jobs/apm-server-schedule-daily.yml:10]:
```suggestion
          default: main
```

File: .ci/jobs/apm-server-schedule-daily.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+- job:
+    name: apm-server-schedule-daily
```
Comment (v1v) [inline .ci/jobs/apm-server-schedule-daily.yml:3]:
```suggestion
    name: apm-server/apm-server-schedule-daily
```

**Comment by v1v** [comment]
I'll merge this now since it's not validated in the CI

**Pull request merged**